python/ray/_private/accelerators/amd_gpu.py

```diff
@@ -6,8 +6,22 @@
 
 logger = logging.getLogger(__name__)
 
-ROCR_VISIBLE_DEVICES_ENV_VAR = "ROCR_VISIBLE_DEVICES"
-NOSET_ROCR_VISIBLE_DEVICES_ENV_VAR = "RAY_EXPERIMENTAL_NOSET_ROCR_VISIBLE_DEVICES"
+if "ROCR_VISIBLE_DEVICES" in os.environ:
+    raise RuntimeError("Please use HIP_VISIBLE_DEVICES instead of ROCR_VISIBLE_DEVICES")
+
+if "HIP_VISIBLE_DEVICES" in os.environ:
+    hip_val = os.environ["HIP_VISIBLE_DEVICES"]
+    if cuda_val := os.environ.get("CUDA_VISIBLE_DEVICES", None):
+        logger.warning(
+            "WARNING: CUDA_VISIBLE_DEVICES: {} and HIP_VISIBLE_DEVICES: {} must be equal.".format(
+                cuda_val, hip_val
+            )
+        )
+    else:
+        os.environ["CUDA_VISIBLE_DEVICES"] = hip_val
```

Comment:
what's the reason to set `CUDA_VISIBLE_DEVICES`? In addition, would you mind using the existing variable instead of hard-coding `CUDA_VISIBLE_DEVICES`?